__tests__/app.test.js

```diff
@@ -0,0 +1,36 @@
+const request = require("supertest")
+const seed = require("../db/seeds/seed")
+
+const testData = require("../db/data/test-data")
+const app = require("../app")
+const db = require("../db/connection")
+
+beforeEach(() => seed(testData))
+
+afterAll(() => db.end())
+describe("GET:/api/topics", () => {
+    test("status: 200 - should resolve with topics array ", () => {
+        return request(app)
+            .get("/api/topics")
+            .expect(200)
+            .then(({ body }) => {
+                const { topics } = body
+                expect(topics).toBeInstanceOf(Array)
+                expect(topics).toHaveLength(3)
+            })
+    })
+    test("status: 200 - each topic should have the correct keys ", () => {
+        return request(app)
+            .get("/api/topics")
+            .expect(200)
+            .then(({ body }) => {
+                const { topics } = body
+
+                topics.forEach((topic) => {
+                    expect(topic).toHaveProperty("slug"), expect.any(String)
+                    expect(topic).toHaveProperty("description"),
+                        expect.any(String)
+                })
```

Comment:
More a personal choice, but I'd argue it makes it cleaner if you were to condense these two tests together 😄